docs/configuration.asciidoc

```diff
@@ -11,7 +11,7 @@ The recommended way to configure Elastic APM for Rails is to create a file `conf
 [source,yaml]
 ----
 ---
-server_url: 'http://localhost:8200'
+server_url: 'https://localhost:8200'
```

Comment:
You sure? We provide a self-signed cert for `localhost`?

Reply:
No we don't provide a cert. 
I wouldn't create an example with a `secret_token` and `http` though.

Reply:
@mikker I removed other changes than setting the `Beta` flag and leave it up to you if you want to change the example in the future.

Reply:
@simitt Thank you!